docs/diagrams/StateSequenceDiagram.puml

```diff
@@ -0,0 +1,47 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Model MODEL_COLOR_T1
+participant ":Model" as Model MODEL_COLOR
+end box
+
+box State #DarkGray
+participant ":State" as State #DimGray
```

Comment:
Is it possible to use constants to represent the State colours like in Model

Reply:
I don't know how to

Reply:
There is a file style.puml that you can edit